docs/extend/extend-cli/cli-developing-a-plugin.md

```diff
@@ -1,219 +1,348 @@
 
-At the end of this tutorial, you will have created a data set diff utility plug-in for Zowe CLI, from which you can pipe
-your plugin's output to a third-party utility for a side-by-side diff of data set member contents.
+## Setting up the new sample plug-in project
+ Download the sample plug-in source and delete the irrelevant content to set up your plug-in project.
+ 
+ Follow these steps:
 
-![Side by Side Diff](../../images/guides/CLI/htmlDiff.png)
+1. Open a terminal and run the command `mkdir zowe-tutorial`.
+    
+    **Note:** All the files created through this tutorial are saved in this tutorial directory.
+2. Enter `cd zowe-tutorial` to change directory into your `zowe-tutorial` folder.
+3. Download the [source code zip file](https://github.com/zowe/zowe-cli-sample-plugin/archive/refs/heads/master.zip) from the Zowe CLI sample plug-in repository.
+4. In your File Explorer, extract the zip file to the  `zowe-tutorial` folder.
+5. Rename the `zowe-cli-sample-plugin-master` directory to `files-util`.
+    
+    This is the project directory used throughout the rest of this tutorial. 
+6. Delete all content within the following folders:
+    - `src/api`
+    - `src/cli`
+    - `docs` folders
+    - `__tests__/__system__/api`
+    - `__tests__/__system__/cli`
+    -  `__tests__/api`
+    - `__tests__/cli`
+7. Return to your terminal and run `cd files-util` to enter the project directory.
+8. Enter `git init` to set up a new Git repository.
+9. Enter `git add --all` to *stage* (track) all files in the current directory with Git.
 
-Completed source for this tutorial can be found on the `develop-a-plugin` branch of the zowe-cli-sample-plugin repository.
+10. Enter `git commit --message "Initial commit"` to save a snapshot of the staged files in your repository.
+11. Run `npm install` to install third-party dependencies defined in the `package.json` file of your Node.js project.
 
-### Cloning the sample plug-in source
- Clone the sample repo, delete the irrelevant source, and create a brand new plug-in. Follow these steps:
+    When successful, a progress bar displays. Once the plug-in is installed, a message displays the status of the packages in the `node_modules` directory.
 
-1. `cd` into your `zowe-tutorial` folder
-2. `git clone https://github.com/zowe/zowe-cli-sample-plugin files-util`
-3. `cd files-util`
-4. Delete the `.git` (hidden) folder.
-5. Delete all content within the `src/api`, `src/cli`, and `docs` folders.
-6. Delete all content within the `__tests__/__system__/api`, `__tests__/__system__/cli`, `__tests__/api`, and `__tests__/cli` folders
-7. `git init`
-8. `git add .`
-9. `git commit -m "initial"`
+    **Note:** If vulnerabilities are found in any of the installed dependencies, refer to [npm Docs](https://docs.npmjs.com/cli/v9/commands/npm-audit) for how to fix them.
```

Comment:
¿maybe instead?: **Note:** If <ins>you are prompted that</ins> vulnerabilities <ins>were found</ins> in any of the installed dependencies, refer to [npm Docs](https://docs.npmjs.com/cli/v9/commands/npm-audit) for how to fix them.

Reply:
Thank you, but I think the original wording is better.

Reply:
Ah. Excellent point. I'll made an update in a (near) future PR and make sure to tag you on it.